phpunit.el

```diff
@@ -62,6 +62,10 @@
                  (function :tag "A function return PHPUnit executable file path.")
                  (string   :tag "PHPUnit command name. (require command in PATH)")))
 
+(defcustom phpunit-args nil
+  "Argument to pass to phpunit command."
+  :type 'string)
```

Comment:
This change has BC break.

You can use [`.dir-locals.el`](https://www.gnu.org/software/emacs/manual/html_node/emacs/Directory-Variables.html) file or `setq-default` in your .emacs file (`init.el`) instead of  `defcustom`/`custom-set-variables`.